fluxc/src/test/java/org/wordpress/android/fluxc/network/UserAgentTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.wordpress.android.fluxc.network
+
+import android.webkit.WebSettings
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mockStatic
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.wordpress.android.util.PackageUtils
+import kotlin.test.assertEquals
+
+private const val APP_NAME = "App Name"
+private const val USER_AGENT = "Default User Agent"
+private const val APP_VERSION = "1.0"
+
+@RunWith(RobolectricTestRunner::class)
+class UserAgentTest {
+    private val context = RuntimeEnvironment.application.applicationContext
```

Comment:
`RuntimeEnvironment.application` is deprecated so we can use `RuntimeEnvironment .getApplication()` instead.

```suggestion
    private val context = RuntimeEnvironment.getApplication().applicationContext
```